src/connect/hooks/tree-view/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './useGetDriveParent';
+export * from './usePathContent';
```

Comment:
I'll add more hooks in a next commit to support common tasks related to the UI state: add a virtual folder, remove UI state of an specific item, update UI state for an specific item, etc.